scalability_and_performance/understanding-node-observability-operator.adoc

```diff
@@ -0,0 +1,325 @@
+----
+
+. Create an `OperatorGroup` object YAML file:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: node-observability-operator
+  namespace: node-observability-operator
+spec:
+  targetNamespaces:
+  - node-observability-operator
+EOF
+----
+
+. Create a `Subscription` object YAML file to subscribe a namespace to an Operator:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: node-observability-operator
+  namespace: node-observability-operator
+spec:
+  channel: alpha
+  name: node-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.Verification
+
+. View the install plan name:
++
+[source,terminal]
+----
+$ oc -n node-observability-operator get sub node-observability-operator -o yaml | yq '.status.installplan.name'
+----
+
++
+.Example output
+[source,terminal]
+----
+install-dt54w
+----
+
+. Verify the install plan status:
++
+[source,terminal]
+----
+$ oc -n node-observability-operator get ip <install_plan_name> -o yaml | yq '.status.phase'
+----
++
+`<install_plan_name>` is the install plan name that you obtained from the output of the previous command.
+
++
+.Example output
+[source,terminal]
+----
+COMPLETE
+----
+
+. Verify that the Node Observability Operator is up and running:
++
+[source,terminal]
+----
+$ oc get deploy -n node-observability-operator
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   UP-TO-DATE  AVAILABLE   AGE
+node-observability-operator-controller-manager  1/1     1           1           40h
+----
+
+=== Installing the Node Observability Operator using the web console
+You can install the Node Observability Operator from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the OpenShift Container Platform web console.
+. In the Administrator's navigation panel, expand *Operators* → *OperatorHub*.
+. In the *All items* field, enter *Node Observability Operator* and select the Node Observability Operator tile.
+. Click *Install*.
+. On the *Install Operator* page:
+.. *Update channel*: Select *alpha*.
+.. *Installation mode*: Select *A specific namespace on the cluster*.
+.. *Installed Namespace*: Choose *node-observability-operator* from the list.
+.. *Update approval*: Select *Automatic*.
+.. Click *Install*.
+
+.Verification
+. In the Administrator's navigation panel, expand *Operators* → *Installed Operators*.
+. Verify that the Node Observability Operator is listed in the Operators list.
+
+== Creating the Node Observability custom resource
+Before you run profiling queries, you must create a `NodeObservability` custom resource (CR).
+
+[IMPORTANT]
```

Comment:
@alebedev87 @sherine-k  Please review and let me know if IMPORTANT block make sense.

Reply:
Yes, it makes sense, thanks for adding this!